manual_jb/content/analyses/commercial_building_recovery.md

```diff
@@ -0,0 +1,88 @@
+`num_samples` <sup>*</sup> | `int` | Samples number | Number of sample scenarios.
+`seed` | `int` | Seed | Initial seed for the probabilistic model.
+`repair_key` | `str` | Repair key | A repair key to use in mapping dataset.
+
+**Input datasets**
+
+key name | type | name | description
+--- | --- | --- | ---
+`buildings` <sup>*</sup> | `ergo:buildingInventoryVer4`<br>`ergo:buildingInventoryVer5`<br>`ergo:buildingInventoryVer6`<br>`ergo:buildingInventoryVer7` | Building dataset |  A building dataset.
+`dfr3_mapping_set` <sup>*</sup> | `incore:dfr3MappingSet` | DFR3 Mapping Set | DFR3 Mapping Set.
+`sample_damage_states` <sup>*</sup> | `incore:sampleDamageState` | Damage states | Sample damage states.
+`mcs_failure` <sup>*</sup> | `incore:failureProbability` | MCS failure | mcs_failure.
+`delay_factors` <sup>*</sup> | `incore:buildingRecoveryFactors` | Delay factors | Delay impeding factors such as post-disaster inspection, insurance claim,<br>and government permit based on building's damage state. Provided by REDi framework.
+
+**Output datasets**
+
+key name | type | parent key              | name | description
+--- | --- |-------------------------| --- | ---
+`time_stepping_recovery` <sup>*</sup> | `incore:buildingRecovery` | Results                 | A dataset containing results (format: CSV)<br>with percentages of commerical building recovery.
+`recovery` <sup>*</sup> | `incore:buildingRecoveryTime` | Building Recovery Time  | A dataset containing results (format: CSV)<br>with commerical building recovery time.
+`total_delay` <sup>*</sup> | `incore:buildingRecoveryDelay` | Building Recovery Delay | A dataset containing results (format: CSV)<br>with commerical building delay time.
+
+<small>(* required)</small>
+
+**Execution**
+
+code snippet:
+
+```
+    # Create Commerical building recovery instance
+    comm_recovery = CommericalBuildingRecovery(client)
+    
+    # Load input building infrastructure dataset
+    comm_recovery.load_remote_input_dataset("buildings", buildings)
+
+    # Load repair mapping
+    repair_service = RepairService(client)
+    mapping_set = MappingSet(repair_service.get_mapping(mapping_id))
+    comm_recovery.set_input_dataset('dfr3_mapping_set', mapping_set)
+    
+    # Load input datasets
+    com_recovery.load_remote_input_dataset("sample_damage_states", sample_damage_states)
+    com_recovery.load_remote_input_dataset("mcs_failure", mcs_failure)
+    com_recovery.load_remote_input_dataset("delay_factors", delay_factors)
+
+    # Specify the result name
+    result_name = "joplin_recovery"
+
+    # Set analysis parameters
+    comm_recovery.set_parameter("result_name", result_name)
+    comm_recovery.set_parameter("seed", seed)
+    comm_recovery.set_parameter("num_samples", 10)
+
+    # Run commerical recovery analysis
+    comm_recovery.run_analysis()
+```
+
+full analysis: [commerical_building_recovery.ipynb](https://github.com/IN-CORE/incore-docs/blob/main/notebooks/commerical_building_recovery.ipynb)
```

Comment:
commercial_building_recovery.ipynb, the link also has the typo, should be commercial not commerical